PhysicsTools/PatAlgos/plugins/PATPhotonProducer.cc

```diff
@@ -370,6 +372,38 @@ void PATPhotonProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSe
     aPhoton.setCryEta( ecalRegData.seedCrysEtaOrX() );
     aPhoton.setIEta( ecalRegData.seedCrysIEtaOrIX() );
     aPhoton.setIPhi( ecalRegData.seedCrysIPhiOrIY() );
+    } else {
+    aPhoton.setEMax(0);
```

Comment:
Was the original change (now moved/added to PATPhotonSlimmer) kept here intentionally or was there a missing commit?